plugins/node/opentelemetry-plugin-dns/test/integrations/dns-lookup.test.ts

```diff
@@ -129,8 +129,8 @@ describe('dns.lookup()', () => {
           hostname,
           forceStatus: {
             code: process.versions.node.startsWith('8')
-              ? CanonicalCode.UNKNOWN
-              : CanonicalCode.INVALID_ARGUMENT,
+              ? StatusCode.UNSET
```

Comment:
I think it should always be set to `StatusCode.Error`.
`CanonicalCode.UNKNOWN` means error.